Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,19 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+{# Possible states are "error", "started", and "done" #}
+<div data-name="notification" data-state="{{ notification_state|default("done") }}">
+    {% set notifications = app.session.flashBag.get("notification") %}
+    {% if notifications|length == 1 %}
+        {{ notifications[0] }}
```

Comment:
You can already have several messages, but with only one state.
And Pjax is synchronous, so there should not be any problem with that AFAICS. However, we should somehow be able to trigger notifications using JS.